src/lib/strings/time.ts

```diff
@@ -61,3 +61,10 @@ export function getAge(birthDate: Date): number {
   }
   return age
 }
+
+export function toSimpleDateString(date: Date) {
+  const mm = date.getMonth() + 1 // 0-indexed
+  const dd = date.getDate()
+  const yyyy = date.getFullYear()
+  return `${yyyy}-${mm}-${dd}`
```

Comment:
This might end up being something like `2024-2-4` which is not universally supported when we feed it to `new Date` below. (E.g. `new Date(`2024-2-4`)` in Safari is `Invalid Date`.